store.go

```diff
@@ -2334,6 +2381,37 @@ func init() {
 	if config.Storage.GraphRoot != "" {
 		DefaultStoreOptions.GraphRoot = config.Storage.GraphRoot
 	}
+	if config.Storage.Options.Devicemapper.ThinPoolDev != "" {
+		DefaultStoreOptions.GraphDriverOptions = append(DefaultStoreOptions.GraphDriverOptions, fmt.Sprintf("dm.thinpooldev=%s", config.Storage.Options.Devicemapper.ThinPoolDev))
+
+	}
+	if config.Storage.Options.Devicemapper.BaseSize != "" {
+		DefaultStoreOptions.GraphDriverOptions = append(DefaultStoreOptions.GraphDriverOptions, fmt.Sprintf("dm.basesize=%s", config.Storage.Options.Devicemapper.BaseSize))
+	}
+	if config.Storage.Options.Devicemapper.Fs != "" {
+		DefaultStoreOptions.GraphDriverOptions = append(DefaultStoreOptions.GraphDriverOptions, fmt.Sprintf("dm.fs=%s", config.Storage.Options.Devicemapper.Fs))
+	}
+	if config.Storage.Options.Devicemapper.MkfsArg != "" {
+		DefaultStoreOptions.GraphDriverOptions = append(DefaultStoreOptions.GraphDriverOptions, fmt.Sprintf("dm.mkfsarg=%s", config.Storage.Options.Devicemapper.MkfsArg))
+	}
+	if config.Storage.Options.Devicemapper.MountOpt != "" {
+		DefaultStoreOptions.GraphDriverOptions = append(DefaultStoreOptions.GraphDriverOptions, fmt.Sprintf("dm.mountopt=%s", config.Storage.Options.Devicemapper.MountOpt))
+	}
+	if config.Storage.Options.Devicemapper.UseDeferredRemoval != "" {
+		DefaultStoreOptions.GraphDriverOptions = append(DefaultStoreOptions.GraphDriverOptions, fmt.Sprintf("dm.mountopt=%s", config.Storage.Options.Devicemapper.UseDeferredRemoval))
+	}
+	if config.Storage.Options.Devicemapper.UseDeferredDeletion != "" {
+		DefaultStoreOptions.GraphDriverOptions = append(DefaultStoreOptions.GraphDriverOptions, fmt.Sprintf("dm.mountopt=%s", config.Storage.Options.Devicemapper.UseDeferredDeletion))
+	}
```

Comment:
Is it possible to have 2 or more dm.mountopt's like: "dm.mountopt=MountOpt dm.mountopt=UseDeferredRemoval dm.mountopt.UseDeferredDeletion" packed into on DefaultStoreOptions.GraphDriverOptions and will things be happy if so?